### PR TITLE
CB-26992 Use latest cdp-infra-tools and its gpg key

### PR DESCRIPTION
**Pull request by Bajzathd**
Azure x86: http://build.eng.hortonworks.com/job/cloudbreak-multi-packer-image-builder-salt-v3/5284/
AWS arm64: http://build.eng.hortonworks.com/job/cloudbreak-multi-packer-image-builder-salt-v3/5285/

### CODE REVIEW

File: saltstack/base/salt/telemetry/init.sls [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+    - name: yum --disablerepo="*" --enablerepo=cdp-infra-tools list available
+
+install_cdp_infra_tools_packages:
+  pkg.installed:
+    - pkgs:
+{% if salt['environ.get']('INCLUDE_CDP_TELEMETRY') == "Yes" %}
+      - cdp-telemetry
+{% endif %}
+{% if salt['environ.get']('INCLUDE_FLUENT') == "Yes" %}
+      - redhat-lsb-core # this will install redhat-lsb-core is required for fluent
```
Comment (TheTinkerDad) [inline saltstack/base/salt/telemetry/init.sls:24]: This line is redundant, redhat-lsb-core is always installed much earlier as it is needed by other stuff too.

**Review by TheTinkerDad — CHANGES_REQUESTED** [review]

File: saltstack/base/salt/telemetry/init.sls (hunk 1 above)
Comment (TheTinkerDad) [reply saltstack/base/salt/telemetry/init.sls:24]: Nevermind, actually. Seems like fluent always needs this while the other components only need it on RHEL 8.

Comment (TheTinkerDad) [reply saltstack/base/salt/telemetry/init.sls:24]: https://github.com/hortonworks/cloudbreak-images/blob/86cca4e9571714bc8a0306ff6c1b71671d7e6a0c/scripts/salt-install.sh#L44

Comment (Bajzathd) [reply saltstack/base/salt/telemetry/init.sls:24]: the earlier install is for rhel8 only

**Review by TheTinkerDad — APPROVED** [review]

**Pull request merged**